### PR TITLE
Implemented starknet_getStorageAt

### PR DESCRIPTION
**Pull request by dpinones**
Implemented starknet_getStorageAt

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

# What is the current behavior?

Issue Number: #362 

# What is the new behavior?

-
-
-

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

# Other information

### CODE REVIEW

**Comment by phklive** [comment]
Hey man how is it going? Let's close the rpc asap.

**Comment by dpinones** [comment]
Hey @phklive, I added the changes that are in the PR #371 to be able to implement the RPC endpoint without Helios errors. The RPC endpoint is functioning properly. However, I am getting an error in the "[check](https://github.com/keep-starknet-strange/beerus/actions/runs/4681351376/jobs/8294010849?pr=369)" job. To merge this PR, the PR #371  where they update the Helios version must be merged first and all jobs must be executed without issues.

File: crates/beerus-rpc/src/api.rs [hunk 1]
```diff
@@ -225,4 +225,11 @@ pub trait BeerusApi {
         request: FunctionCall,
         block_number: u64,
     ) -> Result<Vec<FieldElement>, Error>;
+
+    #[method(name = "getStorageAt")]
+    async fn get_storage_at(
+        &self,
+        contract_address: String,
```
Comment (phklive) [inline crates/beerus-rpc/src/api.rs:232]: If I am not mistaken starknet_getStorageAt also takes a BlockId:
SCR-20230413-qbqi

As seen here.

**Review by phklive — CHANGES_REQUESTED** [review]
Only thing would be BlockId as params rest lgtm

File: crates/beerus-rpc/src/api.rs (hunk 1 above)
Comment (phklive) [reply crates/beerus-rpc/src/api.rs:232]: I agree i will create an issue and take care of the get_nonce now.

Comment (phklive) [reply crates/beerus-rpc/src/api.rs:232]: Will create issues for adding blockId as a parameter after merging BlockId PR: https://github.com/keep-starknet-strange/beerus/pull/391

Well done.

**Comment by phklive** [comment]
lgmt

**Comment by phklive** [comment]
Let's build 🚀

**Pull request merged**